client/landing/stepper/declarative-flow/internals/steps-repository/create-site/index.tsx

```diff
@@ -67,7 +66,6 @@ const CreateSite: Step = function CreateSite( { navigation, flow, data } ) {
 		siteUrl,
 		progress,
 		partnerBundle,
-		siteGoals,
```

Comment:
Can we also remove the `siteGoals: select( ONBOARD_STORE ).getGoals()` line inside the `useSelect` ?

Reply:
This looks like a missed opportunity.

Reply:
Cleaning it up in https://github.com/Automattic/wp-calypso/pull/101921